doc/source/analysis.rst

```diff
@@ -148,6 +148,17 @@ Let's tell runsample to put our analysis into a directory called 947 and also te
 
 You can see from the output that the sample started and finished. If there were errors, they would show up in between those two lines and you would have to view the :doc:`help` documentation.
 
+Run with fasta files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Simply run runsample with the --fasta argument: 
+
+.. code-block:: bash
+
+     $> runsample -od 947 --fasta path/to/fasta/files ngs_mapper/tests/fixtures/functional/947.ref.fasta 947 
```

Comment:
I'm not sure exactly how to do it without effecting the functional tests, but the `path/to/fasta/files` should be an actual path so that the user can run the command as they are reading the documentation.
My first thought was to maybe convert some read files into fasta that we have for the 947 sample...just sounds hard though so not sure what to do.
Maybe add a 3rd fixture that contains only fasta files that is very simple.
Take a small reference file and then duplicate that reference like 100 times and cut it up
```
>myref
ABCDEFGHIJKLMNOPQRSTUVWXYZ

>read1
ABCD
>read2
EFGH
...
```